build_config/lib/src/build_config.g.dart

```diff
@@ -6,7 +6,7 @@ part of 'build_config.dart';
 // JsonSerializableGenerator
 // **************************************************************************
 
-BuildConfig _$BuildConfigFromJson(Map json) {
+BuildConfig _$BuildConfigFromJson(Map<dynamic, dynamic> json) {
```

Comment:
You had to manually update this ya? That is a bit concerning... I can't remember if we ever got the support to add ignore comments back

Reply:
oh yeah, I wasn't paying attention to realize I was in a generated file here and I completely forgot that the reason I never got back to this was that we are blocked by `json_serializable`.

Hopefully if I properly link the issue and close out this PR I won't forget again 😜 

https://github.com/google/json_serializable.dart/issues/517